mitodl/mitxonline
# Versioning of certificate template

#### Pre-Flight checklist


#### What are the relevant tickets?
#887 

#### How should this be manually tested?

- Get or Create two different users
- Enroll them both in any of one course
- The course should have the certificate page in CMS
- Generate certificate for one user and check that user’s certificate.
- Go to the CMS and make some changes in certificate page of that course (e.g. add/remove one or more signatories)
- Generate certificate for second user
- Verify that both users have different certificate template
- One who got the certificate first will have different certificate than a user who got the certificate after making changes in certificate template

#### Screenshots (if appropriate)
<img width="1066" alt="Screenshot 2022-08-29 at 2 59 26 PM" src="https://user-images.githubusercontent.com/77275478/187182950-91739bea-dc5e-4a4b-b613-7a15e80bb565.png">
<img width="1066" alt="Screenshot 2022-08-29 at 3 10 11 PM" src="https://user-images.githubusercontent.com/77275478/187182972-f061f215-0124-4498-a3c2-096b08d01ab3.png">
<img width="1066" alt="Screenshot 2022-08-29 at 3 11 33 PM" src="https://user-images.githubusercontent.com/77275478/187182977-63925d77-504b-4896-a87e-45ebc5798f77.png">
